src/components/Booking/Booking.jsx

```diff
@@ -7,7 +7,7 @@ import "./Booking.scss";
 
 const Booking = (props) => {
   // prettier-ignore
-  const { id, title, firstName, surname, email, roomId, checkInDate, checkOutDate } = props.booking;
```

Comment:
I suggest keeping the surname the same. It matches how we've done things in other parts of the app and stays true to the original data. This way, everything stays consistent and follows the same pattern.